ontology/yaml/resources/METERS/entity_types/HM.yaml

```diff
@@ -48,6 +48,9 @@ HM_CHW_WAC:
   is_canonical: true
   uses:
   - water_volume_accumulator
+  opt_uses:
+  - exchange_thermal_temperature_sensor
```

Comment:
What is the context for this? exhange thermal?

Reply:
It is actually a temperature difference between supply water and return water in chilled water pipeline.
Can it be differential temperature sensor

Reply:
Did the changes